### PR TITLE
fix: Make clear to use script generator in aikar's flags explanation page

### PR DESCRIPTION
**Pull request by Leguan16**

### CODE REVIEW

File: docs/paper/admin/how-to/aikars-flags.md [hunk 1]
```diff
@@ -7,21 +7,15 @@ description: Aikar's flags are a set of JVM flags designed to improve the perfor
 
 ## Recommended JVM startup flags
 
-Use these flags exactly, only changing `Xmx` and `Xms`. These flags work and scale accordingly to any
-size of memory.
+:::warning[Script Generator]
 
-For an automated script to generate these flags for you, see
-our [Script Generator](/misc/tools/start-script-gen).
+**This page only serves as an explanation page.** If you want to generate a start script please visit
```
Comment (zlataovce) [inline docs/paper/admin/how-to/aikars-flags.md:12]:
```suggestion
**This page only serves as an explanation page.** If you want to generate a start script, please visit
```

File: docs/paper/admin/how-to/aikars-flags.md [hunk 2]
```diff
@@ -7,21 +7,15 @@ description: Aikar's flags are a set of JVM flags designed to improve the perfor
...
--XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4
--XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90
--XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem
--XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs
--Daikars.new.flags=true -jar paper.jar --nogui
+java -Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200                              -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch                              -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M                              -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4                              -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90                              -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem                              -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs                              Daikars.new.flags=true -jar paper.jar --nogui
```
Comment (zlataovce) [inline docs/paper/admin/how-to/aikars-flags.md:18]: what's up with these spaces?

Comment (Leguan16) [reply docs/paper/admin/how-to/aikars-flags.md:18]: Is it not what ollie wanted? https://canary.discord.com/channels/289587909051416579/947676006964162611/1239256556919455764

Comment (zlataovce) [reply docs/paper/admin/how-to/aikars-flags.md:18]: I don't think the huge empty spaces should be there
```suggestion
java -Xms10G -Xmx10G -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs -Daikars.new.flags=true -jar paper.jar --nogui
```

Comment (Leguan16) [reply docs/paper/admin/how-to/aikars-flags.md:18]: now i understand what he mean lol I was confused as well but i thought "if ollie wants it" xd

Comment (Leguan16) [reply docs/paper/admin/how-to/aikars-flags.md]: fixed haha

**Review by Machine-Maker — APPROVED** [review]

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**